examples/connparams/connparams.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/nlopes/slack"
+)
+
+func main() {
+	api := slack.New(
+		"YOUR TOKEN HERE",
+		slack.OptionDebug(true),
+		slack.OptionLog(log.New(os.Stdout, "slack-bot: ", log.Lshortfile|log.LstdFlags)),
+	)
+
+	// turn on the batch_presence_aware option
+	rtm := api.NewRTM(slack.RTMOptionConnParams(map[string]string{
```

Comment:
I believe you want [url.Values](https://golang.org/pkg/net/url/#Values) here.

Reply:
Thanks! yep that express it better - updated.